shared/src/common/types/user/update-user.ts

```diff
@@ -0,0 +1,24 @@
+import { UserSex } from './user.enum';
+
+interface UserFields {
+  sex: UserSex;
+  birthYear: string;
+  location: string;
+  name: string;
+  phone: string;
+  email: string;
+}
+
+interface ProfileFieldsRequestData extends UserFields {
+  password: string;
+  newPassword: string;
+  repeatNewPassword: string;
+}
+
+interface ProfileFieldsResponseData extends UserFields {}
```

Comment:
Empty interfaces that extend smth should `type`s
```suggestion
type ProfileFieldsResponseData = UserFields;
```